src/services/codefixes/fixSpelling.ts

```diff
@@ -57,6 +57,11 @@ namespace ts.codefix {
             }
             suggestedSymbol = checker.getSuggestedSymbolForNonexistentProperty(node, containingType);
         }
+        else if (isBinaryExpression(parent) && parent.operatorToken.kind === SyntaxKind.InKeyword && isPrivateIdentifier(parent.left) && parent.left === node) {

```

Comment:
If you change the order here, you don't need the `Debug.assert` on line 62.

```suggestion
        else if (isBinaryExpression(parent) && parent.operatorToken.kind === SyntaxKind.InKeyword && parent.left === node && isPrivateIdentifier(node)) {
```

Reply:
Nice spot